examples/client/object_server.c

```diff
@@ -224,6 +224,50 @@ static uint8_t prv_get_value(lwm2m_data_t * dataP,
     }
 }
 
+static void remove_optional_resource(uint16_t *resList, int val, int id, int *nbRes) {
```

Comment:
If we want to follow trough with constification:
```suggestion
static void remove_optional_resource(uint16_t *const resList, const int val, const int id, int *const nbRes) {
```